rhc-canonical-facts.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=rhc canonical-facts service
+Documentation=https://github.com/RedHatInsights/rhc
+
+[Service]
+Type=oneshot
+User=root
+Group=yggdrasil-worker
+ExecStart=rhc canonical-facts
+StandardOutput=truncate:/var/lib/yggdrasil/canonical-facts.json
+StandardError=journal
+UMask=0020
```

Comment:
This umask is not correct. There should be: `UMask=0027`. Otherwise, non-root user will not only be able to read such file, but they will be able to write to such file as you can see:

```
root@localhost:/# ls -l /var/lib/yggdrasil/canonical-facts.json 
-rw-r--rw-. 1 root root 782 Sep 18 12:04 /var/lib/yggdrasil/canonical-facts.json
```